dwh/indicators.py

```diff
@@ -107,6 +123,8 @@ def find_date_range(args: argparse.Namespace) -> Tuple[str, str, str, str, str]:
       quarterly_start_str) = find_date_range(args)
   print('Source directory: {0}'.format(args.src_dir))
   print('Date range:  {0} - {1}'.format(start_date, end_date))
+  print('Location Id: {0}'.format(args.location_ids))
```

Comment:
nit: `Id` -> `Ids` since this can be a list? (also next line: `type` -> `types`).

Reply:
sure, thanks for the heads up!